### PR TITLE
Nodejs support

### PR DESCRIPTION
**Pull request by brettmc**
Summary
* add babel-plugin-transform-runtime to allow build to run in node.js
* add websocket to index.js for ease of access
* update readme with example for using the client from node.js

Test Information

This PR was tested on:
* docker node:alpine (node 8.2.1 + 8.4.0)
* docker node:latest (debian/jessie node 8.4.0)

### CODE REVIEW

**Comment by brettmc** [comment]
FYI the unknown with this PR is whether the use of the `transform-runtime` plugin has any adverse affects downstream (due to the code being transpiled differently). I don't know how to test for that....

File: .babelrc [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "plugins": [
     ["module-resolver", {
       "root": ["./src", "."]
-    }]
+    }],
+    "transform-runtime"
```
Comment (enahum) [inline .babelrc]: Thia should be taken care from the app that is going to implement and use this library if the sources want to be used

File: src/client/index.js [hunk 2]
```diff
@@ -3,11 +3,13 @@
...
+import WebSocketClient from './websocket_client.js';
 
 const Client = new ClientClass();
 const Client4 = new ClientClass4();
 
 export {
     Client,
-    Client4
+    Client4,
+    WebSocketClient
```
Comment (enahum) [inline src/client/index.js]: Why do we want this here?

**Comment by brettmc** [comment]
I got this working with no code changes - `babel-polyfill` was the missing ingredient.

**Comment by jwilander** [comment]
@enahum if it looks good to you LGTM

**Pull request merged**